lib/mongoose_push_behaviour.ex

```diff
@@ -0,0 +1,5 @@
+defmodule MongoosePushBehaviour do
```

Comment:
This shouldn't be a root level module, no module should be (I know we already have one, that was a mistake that I intend to fix soon).

Reply:
done